### ISSUE TITLE
Confusion about the experiments of MFCCA model

### ISSUE THREAD

**Issue by yjiangling** [alibaba-damo-academy/FunASR#443]
Hi, I'm reading the paper of "MFCCA:MULTI-FRAME CROSS-CHANNEL ATTENTION FOR MULTI-SPEAKER ASR IN MULTI-PARTY MEETING SCENARIO" recently, and meet a few questions. I would be very grateful if you could spare some time to answer my question.

![1](https://user-images.githubusercontent.com/42861880/235119086-52eb79b2-d4cb-4d53-b5e1-36c042873be7.PNG)

1. The four baseline models and MFCCA models use Eval-Ali and Test-Ali for test? Or other test dataset? Because I checked all samples of the Eval-Ali and Test-Ali, there are only 1 channel or 8 channel waves, but the table showed 1-ch、2-ch、4-ch、6-ch、8-ch are tested, how to get 2-ch、4-ch and 6-ch waves?

2. The first baseline model -- Single channel model saied **"use the first channel of Train-Ali-far for training and testing"**, I'm confused this model do not use the same test dataset as other models? What's more, the model trained with one channel, how to test with 2/4/6/8-ch samples? Use the first channel of them as the third baseline model (Random selection) did?

3. The similar proble for the fourth baseline model -- Complex convolution, it was trained with 8 channels samples, how to test with 1/2/4/6-ch samples?

4. How to get one channel output at the end of the Encoder for the Multi-frame cross-channel model with no Convolution fusion?

Besides, there is a small question about data processing that I would like to ask you. As introduces in ModelScope Forum, the train data would be like this for MFCCA model fine-tuning after processing with the gudience of AliMeeting:

![2](https://user-images.githubusercontent.com/42861880/235119157-f1c6aea3-fd5f-4f97-b404-71a165412b48.PNG)

But in fact, the label of wave in text file have no blank in each word and the symbol is "$" instaed of "src” between two speakers. What' s wrong? 

![3](https://user-images.githubusercontent.com/42861880/235119197-c04580d0-84c9-4fb0-b35c-2a2a299d35e4.PNG)

Thanks a lot in advance!!!

**Comment by yufan-aslp** [comment]
1. only use far-filed Eval-Ali and Test-Ali for test.  1-ch test set select the 1st channel of the 8-ch far-filed Eval-Ali and Test-Ali test sets. 2-ch test set select the 1st and 5th channel of the 8-ch far-filed Eval-Ali and Test-Ali test sets. 4-ch test set select the 1st, 3rd, 5th and 7th channel of the 8-ch far-filed Eval-Ali and Test-Ali test sets. 6-ch test set select the 1st,2nd, 3rd, 5th, 6th and 7th channel of the 8-ch far-filed Eval-Ali and Test-Ali test sets. 
2. First baseline model uses the first channel of  the 2/4/6/8-ch samples, so the results are same. For training, the first baseline model (Single channel model) only uses the 1st channel of  the 8-ch far-filed Eval-Ali and Test-Ali test sets. But, the third baseline model (Random selection) randomly selects a channel of the 8-ch far-filed Eval-Ali and Test-Ali test sets for training. For Testing, the both use the 1st channel of  the 2/4/6/8-ch samples.
3. As explained in the Convolution fusion section 3.3 of our paper, the number of input channels in the convolution module is fixed. Therefore, if the channel number of the input is less than the pre-configured value, we need to expand the channel by simple repeating.
4. Concat or Mean
5.  "$" and "src” is both ok. ModelScope can not output "$", or any other symbols. So, we replace the "$" with "src” in "tokens.txt".

**Comment by yjiangling** [comment]
> 

Oh，got it, thanks a lot for the detail explication， thank you very much!!!
